cms/templates/admin/cms/page/tree/menu.html

```diff
@@ -17,8 +17,8 @@
     {% endblock %}
     "
     {% if is_popup %}onclick="opener.dismissRelatedLookupPopup(window, {{ page.id }}); return false;"{% endif %}
-    data-id="{{ page.pk }}"
-    data-node-id="{{ node.pk }}"
+    data-id="{{ page.pk|unlocalize }}"
```

Comment:
I would love to get other peoples opinion about `unlocalize` vs. `safe`

Reply:
AFAICS they have a completely different scope and the output of either filter is different: `unlocalize` will force Django not to localize the value, i.e. by not applying THOUSAND_SEPARATOR in this case, while `safe` instructs Django that it doesn't need to html-escape the passed variable.  So filtering through `safe` in this case would *not* resolve the issue because at best we'd get  `1´000` instead of `1&apos;000` in the back-end, both being unparseable when passed directly to the ORM.

Reply:
Absolutely (that's why it may be called a hack). The process, however, is that first the number is marked as "safe" and with this marker later **not converted** (neither by escaping html nor by number formatting). The `1&apos;000` situation should not happen. However, I am happy to go with the `unlocalized` version as it has more meaning.